### PR TITLE
docs: Add instruction for components images update

### PR DESCRIPTION
**Pull request by kkujawa-sumo**

### CODE REVIEW

File: docs/dev/update.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+1. Replace old version of watches.yaml with the new version:
+
+   ```bash
+   mv watches_new.yaml watches.yaml
+   ```
+
+1. Prepare the commit with component images update.
+
+[container_cerification]: https://github.com/SumoLogic/sumologic-openshift-images?tab=readme-ov-file#container-certification
```
Comment (sumo-drosiek) [inline docs/dev/update.md:89]:
```suggestion
[container_cerification]: https://github.com/SumoLogic/sumologic-openshift-images/blob/main/README.md#container-certification
```

I would point directly to README

File: docs/dev/update.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+1. Build and certify components container images, make steps from [this][container_cerification] list.
+
+1. Save output of `make verify` from [sumologic-openshift-images][sumologic-openshift-images] to the file, e.g.
+
+   ```bash
+   make verify > images.txt
+   ```
+
+1. Check the content of `images.txt` and make sure that all images are available - no `MISSING` entries in the `images.txt`.
```
Comment (sumo-drosiek) [inline docs/dev/update.md:15]: We could provide `grep` command here

File: docs/dev/update.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+1. Check the content of `images.txt` and remove duplicated entries for single component (multiple versions of container images for single component).
```
Comment (sumo-drosiek) [inline docs/dev/update.md:17]: We could provide combination of `uniq`, `sort` and `grep` but not sure if its worth to do it

Comment (kkujawa-sumo) [reply docs/dev/update.md:17]: The content in images.txt is sorted, it is not trivial command to find 4 entries when 2 entries are ok 🤔 
I would leave it as it is.

File: docs/dev/update.md (hunk 2 above)
Comment (kkujawa-sumo) [reply docs/dev/update.md]: changed

**Review by sumo-drosiek — APPROVED** [review]

**Pull request merged**